src/main/java/com/backbase/ct/bbfuel/dto/entitlement/JobProfile.java

```diff
@@ -23,6 +23,7 @@
     private List<String> roles;
     private Boolean isRetail;
     private List<Permission> permissions;
+    private String type;
```

Comment:
Why `String` and not `FunctionGroupBase.Type`?

Reply:
Because in the payload  for creating job profile for type it is expected String.